apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx

```diff
@@ -65,7 +80,45 @@ interface LifeEventsSectionProps {
 const LifeEventsCardsSection: React.FC<LifeEventsSectionProps> = ({
   title = 'Lífsviðburðir',
 }) => {
-  return (
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
```

Comment:
is this something we would want to create a custom hook for?

Reply:
Good call. I've created that.

Reply:
No, I change my mind. It's too messy to conditionally render something with SSR. The current strategy is now to render **both** version and hide either one with CSS.